spinn_front_end_common/utility_models/reverse_ip_tag_multicast_source_machine_vertex.py

```diff
@@ -673,16 +704,6 @@ def is_recording(self):
     })
     def update_buffer(
             self, first_machine_time_step, run_until_timesteps):
-        """ Updates the buffers on specification of the first machine timestep.
-            Note: This is called by injection.
-
-        :param first_machine_time_step:\
-            The first machine time step in the simulation
-        :type first_machine_time_step: int
-        :param run_until_timesteps:\
-            The last machine time step in the simulation
-        :type run_until_timesteps: int
-        """
```

Comment:
I think this was removed by the merge for some reason...

Reply:
im passing this one

Reply:
Looks like a merge issue, resolved.